File: Python Programming/problems on string manipulation/problem115.py
'''
115. Write a program which 2 strings from user and concat second string after first string. (Implement strcat() function). 
	
	Input :   “Marvellous Infosystems”    “Logic Building” 
	Output :  “Marvellous Infosystems Logic Building” 
	
	Author : Milind Nalawade
	
	Date : 16/07/2021
		
	Language : Python 3.9.0
	
'''
#====
#Logic
#====
def StringCon(str1,str2):
	return str1+str2
#=============
#entry point function
#=============
def main():
	str1=input("\n Enter your string1 : ")
	str2=input("\n Enter your string2 : ")
	str3=""
	str3=StringCon(str1,str2)
	print("\n Output : {}\n".format(str3) )
#=====
#starter
#=====
if __name__=="__main__":
	main()

File: C Programming/problems on strings maipulation/problem115.c
/*
115. Write a program which 2 strings from user and concat second string after first string. (Implement strcat() function). 
	
	Input :   “Marvellous Infosystems”    “Logic Building” 
	Output :  “Marvellous Infosystems Logic Building” 
	
	Author : Milind Nalawade
	
	Date : 16/07/2021
	
	Language : C programming
	
*/

//Logic
#include<stdio.h>
#include<string.h>

//Logic
void StrCon(char* arr,char* brr,char* crr)
{
	
	while(*arr!='\0')
	{
		*crr=*arr;
		crr++;
		arr++;
	}
	while(*brr!='\0')
	{
		*crr=*brr;
		crr++;
		brr++;
	}
	*crr='\0';

}

//Entry point function
int main()	
{
	char arr[30];
	char brr[30];
	char crr[60];
	printf("\n Enter your string1 : ");
	scanf("%[^'\n']s",&arr);
	printf("\n Enter your string2 : ");
	scanf(" %[^'\n']s",&brr);
	StrCon(arr,brr,crr);
	printf("\n Output : %s\n",crr);
	return 0;
}